src/components/MyReservationsPage.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import styles from "./MyReservationsPage.module.css";
+
+const MyReservationsPage = () => {
```

Comment:
- [ ] OPTIONAL: When I scroll through the carousel, the more I scroll, the more reservation options disappear on "Reservations".  I made a small loom video to show you this strange bug:

https://www.loom.com/share/ef7200efd9e94a07a4ee5903f73d278f

Kindly take a look and fix this bug, so your app works as intended.

This is OPTIONAL because it wasn't mentioned in the last review, however, I highly suggest you fix it to ensure success in your final evaluation.

Reply:
Due to time restrictions, we decide to review the project later and work on fixing this good suggestion you have made. Thanks for noticing that.